packages/cli/src/cli.ts

```diff
@@ -52,6 +53,14 @@ program
         await promptNewIntegration(path.resolve(process.cwd(), dirPath));
     });
 
+program
+    .command('dev')
+    .description('run the integrations dev server')
+    .option('-p, --port <number>', 'port to run the dev server on')
```

Comment:
What port/server ? why should the end user know this?

It feels like I'll have to access this port, when it's not true. Why not use a random port?